### PR TITLE
Initial Russian translation

### PR DESCRIPTION
**Pull request by DeeKey**
Please note, that I have added a direct link to assembly in Russian how to buy and assemble  the sensor. It is maintained by me in GoogleDocs. 
    "docs": "<a href=\"shorturl.at/TY014\"></a>",

### CODE REVIEW

File: src/components/LanguageSwitcher.svelte [hunk 1]
```diff
@@ -43,5 +43,8 @@
         <a on:click={menuToggle} href="{`sk/${pathWithoutLang}`}"
            class="uppercase block px-2 py-1 text-gray-700 font-semibold rounded hover:bg-teal-500 hover:text-white"
            class:selected="{lang === 'sk' ? 'selected' : ''}"> {flag('sk')} sk</a>
+        <a on:click={menuToggle} href="{`ru/${pathWithoutLang}`}"
+           class="uppercase block px-2 py-1 text-gray-700 font-semibold rounded hover:bg-teal-500 hover:text-white"
+           class:selected="{lang === 'ru' ? 'selected' : ''}"> {flag('ru')} sk</a>
```
Comment (ohheyitsdave) [inline src/components/LanguageSwitcher.svelte]: you forgot to change `sk` to `ru`.

File: src/utils/initI18n.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import en from '../../locales/en';
 import de from '../../locales/de';
 import fr from '../../locales/fr';
 import sk from '../../locales/sk';
+import sk from '../../locales/ru';
```
Comment (ohheyitsdave) [inline src/utils/initI18n.js]: like in the other comment `sk`to `ru`

**Review by ohheyitsdave — CHANGES_REQUESTED** [review]
Thanks @DeeKey for your contribution 😊 
Just two little errors, then I can merge it to develop branch.

**Comment by DeeKey** [comment]
Thank you for your attention!
Over time there will definitely be more. Hard to commit everything in one time without making any mistake  (((
Do you need any input from me, or you will fix everything yourself?

**Review by ohheyitsdave — APPROVED** [review]
hope this works fixing the issues inside Github.

**Comment by ohheyitsdave** [comment]
Works 🙌 Thanks again for your help!

**Pull request merged**